ddtrace/_trace/utils_botocore/span_pointers/telemetry.py

```diff
@@ -0,0 +1,18 @@
+from ddtrace._trace.telemetry import record_span_pointer_calculation as base_record_span_pointer_calculation
```

Comment:
What's the reason for defining the actual methods in ddtrace/_trace and then using them here under the same name?

Reply:
Oh and to handle how to add the additional_tags. Ok!